btree/btree.c

```diff
@@ -4344,15 +4492,14 @@ static int64_t btree_del_tick(struct m0_sm_op *smop)
 		int64_t    nxt = P_FREENODE;
 
 		lev = &oi->i_level[oi->i_used];
-		if (lev->l_freenode) {
+		if (lev->l_node->n_free) {
```

Comment:
As mentioned above you cannot use this n_free variable. Re-design this code.

Reply:
done